proto/greptime/v1/common.proto

```diff
@@ -97,10 +97,32 @@ enum ColumnDataType {
   DURATION_MILLISECOND = 27;
   DURATION_MICROSECOND = 28;
   DURATION_NANOSECOND = 29;
+  DECIMAL_128 = 30;
 }
 
 message IntervalMonthDayNano {
   int32 months = 1;
   int32 days = 2;
   int64 nanoseconds = 3;
 }
+
+// (hi: high 64 bits, lo: low 64 bits, precision, scale) are used to
+// construct the decimal128 value.
+message Decimal128 {
+  int64 hi = 1;
+  int64 lo = 2;
+  int32 precision = 3;
```

Comment:
Here, the precision and scale of each decimal128 are saved, and this design will be redundant for `Column`, because the relevant information has been saved in `ColumnDataTypeExtension`. 
Or implement two kinds of decimal128, one for `Row` (requires precision and scale) and the other for Column (not required). 
Will this design have any impact on the future?

Reply:
We have `schema` in `Rows`, but I'm not sure whether we should store this in a `Value`.

Reply:
We can still use `Decimal128` and remove precision and scale from it. We can add it later if we find it necessary.

Reply:
OK，I will remove them